### PR TITLE
Update Partner Contact Personal Information module to 9.0

### PR DESCRIPTION
**Pull request by richard-willdooit**
partner_contact_personal_information updated to version 9.0

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,8 +42,10 @@ coverage.xml
 # Translations
 *.mo
 
-# Pycharm
-.idea
```
Comment (pedrobaeza) [inline .gitignore:46]: Don't remove this one

File: partner_contact_personal_information_page/README.rst [hunk 2]
```diff
@@ -27,11 +34,27 @@ For further information, please visit:
...
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+partner-contact/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+partner-contact/issues/new?body=module:%20
+partner_contact_personal_information_page%0Aversion:%20
+9.0.1.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline partner_contact_personal_information_page/README.rst]: Put only 9.0

File: .gitignore (hunk 1 above)
Comment (richard-willdooit) [inline .gitignore:46]: Hmm, someone else told me to remove it!  I agree with you.

File: .gitignore [hunk 3]
```diff
@@ -45,6 +45,11 @@ coverage.xml
 # Pycharm
```
Comment (elicoidal) [inline .gitignore:45]: remove this file

File: partner_contact_personal_information_page/README.rst [hunk 4]
```diff
@@ -42,6 +65,7 @@ Contributors
 * Jairo Llopis <j.llopis@grupoesoc.es>
 * Matjaž Mozetič <m.mozetic@matmoz.si>
 * Rudolf Schnapka <schnapkar@golive-saar.de>
+* Richard deMeester <richard@willowit.com.au>
 
 Maintainer
```
Comment (elicoidal) [inline partner_contact_personal_information_page/README.rst:72]: update with latest description

File: .gitignore (hunk 3 above)
Comment (richard-willdooit) [inline .gitignore:45]: @elicoidal @pedrobaeza 

?  I just got asked to not remove this.  Please decide  between you whether to remove or not.

File: partner_contact_personal_information_page/README.rst (hunk 4 above)
Comment (richard-willdooit) [inline partner_contact_personal_information_page/README.rst:72]: Not sure what you mean here.

File: .gitignore (hunk 3 above)
Comment (elicoidal) [inline .gitignore:45]: Oups: sorry.
usually we do not keep the git files in the modules. If @pedrobaeza has 
good reason to keep it then keep it.
Sorry for the noise

File: partner_contact_personal_information_page/README.rst (hunk 4 above)
Comment (elicoidal) [inline partner_contact_personal_information_page/README.rst:72]: Please check the part Maintainer in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: .gitignore (hunk 3 above)
Comment (richard-willdooit) [inline .gitignore:45]: It is not actually in the module, but in the root.  It was included to extend the ignored files, without doing a pull request just to update .gitignore.

Comment (elicoidal) [inline .gitignore:45]: OK

File: partner_contact_personal_information_page/README.rst (hunk 4 above)
Comment (richard-willdooit) [inline partner_contact_personal_information_page/README.rst:72]: Now you are really doing my head in - I can't see what part of maintainer is wrong?

Comment (elicoidal) [inline partner_contact_personal_information_page/README.rst:72]: SorryI got tricked by github...
Folded content :/

**Comment by pedrobaeza** [comment]
Now that I review the module, I think there's a best way to achieve this: declare this view on each module taking as xml-id one with the prefix `base.`. This way, any module will be independent, but if several modules are installed at the same time, there's no duplication of the view.

**Comment by gurneyalex** [comment]
:+1:

@pedrobaeza do you want the module fixed or would you rather move forward?

**Comment by pedrobaeza** [comment]
Well, as this is a migration to v9, I would prefer to have it solved from the beginning and not put dependencies that we have to keep all the v9 life.

**Comment by richard-willdooit** [comment]
OK - on it!

**Comment by gurneyalex** [comment]
@pedrobaeza is the updated version ok with you?

**Comment by richard-willdooit** [comment]
Can this be merged???

**Comment by moylop260** [comment]
:+1:

**Comment by elicoidal** [comment]
:+1:

**Pull request merged**